README.md

```diff
@@ -150,6 +151,13 @@ We have a badge ! Link it to your live Storybook example.
 [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](link to site)
 ```
 
+## Community
+
+-   Tweeting via [@storybookjs](https://twitter.com/storybookjs)
+-   Blogging at [Medium](https://medium.com/storybookjs)
+-   Chatting on [Slack](https://now-examples-slackin-rrirkqohko.now.sh/)
```

Comment:
Let's use Discord here.

Reply:
Sure, have the URL please?

Reply:
https://discord.gg/sMFvFsG

Reply:
My bad. I meant, add discord here. Let's have both slack and discord.